Add humidity from external sensor

LtHummus opened the pull request:
This PR adds the ability to read the humidity from an external sensor (namely PAC-USWHS003-TH-1). There should not be any additional configuration needed to make this work, though it will only work if direct access is enabled. 

I only was able to do hacky testing of this for those without temperature sensors, because all of my units have sensors attached to them.

fjs21 reviewed (changes requested):
I know it is probably just window dressing but can you correct the spelling from 'Humdity' to 'Humidity'

LtHummus commented:
Ha, good catch! (No one has ever accused me of being an accurate speller ;) ). I’ll fix that up tonight (and some of the things called out by the linter)

Jason-Morcos commented on src/ductless.ts:
```diff
@@ -306,12 +333,40 @@ export class KumoPlatformAccessory_ductless {
+      this.platform.log.debug('setting humidity to %s', currentValue);
+
+      if (ourSensor.battery) {
+        this.Humdity.updateCharacteristic(this.platform.Characteristic.StatusLowBattery, ourSensor.battery < 10);
```
Would it be possible to add a battery level characteristic as well, in addition to the this.platform.Characteristic.StatusLowBattery characteristic? I believe there's one in the spec

LtHummus replied:
I looked for one, but I only saw a low battery alert: https://developers.homebridge.io/#/service/HumiditySensor

Jason-Morcos replied:
I think it might be this? https://developers.homebridge.io/#/service/Battery

LtHummus replied:
Ahhh good call. I'll experiment with that tonight and see what I can make.

LtHummus replied:
Got this working...added a battery service and it seems to work as expected.

![image](https://user-images.githubusercontent.com/1617612/176809810-641618c5-265e-4764-b34b-490d475597e0.png)
![image](https://user-images.githubusercontent.com/1617612/176809841-88d32bfc-255e-4c64-91b6-c733719b58e9.png)

Jason-Morcos reviewed (commented):
This looks super cool, @LtHummus!

LtHummus commented:
- Spelled `Humidity` correctly :)
- Fixed complaints from the linter (I usually write in Go at work, so you can see that seep in ;) )
- Added support to set the battery state

Jason-Morcos commented:
Hi @fjs21! Do you have an estimated release date for a release that includes these changes? Is there a way to/would it be helpful to beta test this change in some way?

Thanks!
